recipes/maxbin2/meta.yaml

```diff
@@ -0,0 +1,28 @@
+build:
+  number: 0
+
+package:
+  name: maxbin2
+  version: "2.2.1"
+source:
+  fn: MaxBin-2.2.1.tar.gz
+  url: http://downloads.jbei.org/data/microbial_communities/MaxBin/getfile.php?MaxBin-2.2.1.tar.gz
+  md5: 3748e8f65b5bf18f8bd341210724cc5c
+requirements:
+  build:
+    - gcc # [linux]
+    - llvm # [osx]
+  run:
+    - libgcc # [linux]
+    - perl
+    - bowtie2
+    - fraggenescan >=1.30
+    - hmmer
+    - idba
+test:
+  commands:
+    - MaxBin > /dev/null
+about:
+  home: "http://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html"
+  license: "BSD 3-clause"
+  summary: "MaxBin is software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm."
```

Comment:
I find useful to add an `extra` category with the recipe maintainer(s) and the associated DOI pointing to the original article (if available). Here is an example: [abawaca](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/abawaca/meta.yaml#L29-L32)